tests/mongo.yml

```diff
@@ -112,7 +112,7 @@ results:
       # https://github.com/FerretDB/FerretDB/issues/2915
       - mongo/jstests/aggregation/use_query_sort.js
     capped:
-      # https://github.com/FerretDB/FerretDB/issues/3458
+      # TODO add link
```

Comment:
Missing link?

Reply:
Yes, I will re-request review once I have added it. I need to run capped tests locally first.

Reply:
done